test-resources/bin/pdb-test-env

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
```

Comment:
@wkalt ok, right - for now I just expanded the usage information.  Just shout if that doesn't address your concerns.